openupgrade_scripts/scripts/base_address_extended/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,36 @@
+---Models in module 'base_address_extended'---
+model res.city (moved from base_address_city)
+# NOTHING TO DO
+
+---Fields in module 'base_address_extended'---
+base_address_extended / res.city                 / __last_update (datetime)      : previously in module base_address_city
+base_address_extended / res.city                 / _order                        : previously in module base_address_city
+base_address_extended / res.city                 / country_id (many2one)         : previously in module base_address_city
+base_address_extended / res.city                 / display_name (char)           : previously in module base_address_city
+base_address_extended / res.city                 / name (char)                   : previously in module base_address_city
+base_address_extended / res.city                 / state_id (many2one)           : previously in module base_address_city
+base_address_extended / res.city                 / zipcode (char)                : previously in module base_address_city
+base_address_extended / res.country              / enforce_cities (boolean)      : previously in module base_address_city
+base_address_extended / res.country              / street_format (text)          : DEL required
```

Comment:
Where is this now handled?

Reply:
It seems that this behavior disappeared. See Fabien's commit description here:

https://github.com/odoo/odoo/commit/5b72f017662525362ff1e1bc8173a7ab395dbcf9

Reply:
Reading the commit message (that you can annotate in this work file for reference), it seems there's now a `address_view_id` that reflects such format, so it's not disappearing at all. I don't think anybody has created custom countries, and I suppose they are provided correctly with the proper `address_view_id`, so it can be nothing to do, but please check for being sure, and reflect all of this in the analysis work file for not having to wonder this in the future.

Reply:
All countries have an empty `address_view_id` by default.
So when Odoo deletes its field `street_format`, we are loosing the information they provided in previous versions.
Is there a way to prevent that? We won't create a custom view for all countries data?

Reply:
Don't rename it then. The rename should be done just in case of conflicting name. If not, it will require a pre-migration script to rename back the column instead of just installing the replacement module.